clar2wasm/src/words/contract.rs

```diff
@@ -128,6 +128,7 @@ impl ComplexWord for ContractCall {
 }
 
 #[cfg(test)]
+#[allow(clippy::expect_used, clippy::unwrap_used)]
```

Comment:
It looks like we can configure clippy to allow these in tests instead of adding the attributes everywhere:

```
allow-unwrap-in-tests = true
allow-expect-in-tests = true
```

Reply:
Added in clippy.toml